core/templates/pages/topic-viewer-page/topic-viewer-page.component.ts

```diff
@@ -67,15 +68,19 @@ export class TopicViewerPageComponent implements OnInit {
     private urlInterpolationService: UrlInterpolationService,
     private urlService: UrlService,
     private windowDimensionsService: WindowDimensionsService,
+    private windowRef: WindowRef,
   ) {}
 
   ngOnInit(): void {
     if (this.urlService.getPathname().endsWith('revision')) {
-      this.setActiveTab('subtopics');
```

Comment:
Why is this removed?
Or, if the function call is not required, then that function can also be removed, right? Since it doesn't seem to be called anywhere else.

Reply:
The function will be called from the html, when the user click to change the tab. Before `setActiveTab` only changed the value of `activeTab`, but now it will also change the url. So replaced `this.setActiveTab(...tabName..)` directly with `this.activeTab = ..tabName..` here.